weblogic-azure-aks/src/main/arm/scripts/buildWLSDockerImage.sh

```diff
@@ -231,18 +226,23 @@ function build_wls_image() {
 export script="${BASH_SOURCE[0]}"
 export scriptDir="$(cd "$(dirname "${script}")" && pwd)"
 
+source ${scriptDir}/common.sh
+
 export wlsImagePath=$1
 export azureACRServer=$2
 export azureACRUserName=$3
 export imageTag=$4
 export appPackageUrls=$5
 export ocrSSOUser=$6
 export wlsClusterSize=$7
+export enableSSL=$8
 
 export acrImagePath="$azureACRServer/aks-wls-images:${imageTag}"
 export ocrLoginServer="container-registry.oracle.com"
-export wdtDownloadURL="https://github.com/oracle/weblogic-deploy-tooling/releases/download/release-1.9.7/weblogic-deploy.zip"
-export witDownloadURL="https://github.com/oracle/weblogic-image-tool/releases/download/release-1.9.11/imagetool.zip"
+export wdtDownloadURL="https://github.com/oracle/weblogic-deploy-tooling/releases/download/release-1.9.14/weblogic-deploy.zip"
```

Comment:
Do we want to hardcode specific versions here or use latest? If we don't use latest, what will be the process to periodically update these files?

Reply:
To clarify, you can download latest at `https://github.com/oracle/weblogic-deploy-tooling/releases/latest/download/weblogic-deploy.zip`

Reply:
Hello @rjeberhard,  @edburns and I discussed the version before, we will use a fixed version in the offer, and update the version periodically with well test.

Reply:
Hello @rjeberhard we strongly prefer to pin to a specific version and commit to updating it with a known and agreed cadence.  We can discuss the cadence at the next status meeting if you like.

Reply:
That's fine, Ed. I just wanted to know that the cadence would exist. It's very easy, to forget about it and have customers hit known bugs fixed ages ago.